sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpConnectionScope.cs

```diff
@@ -232,8 +235,8 @@ public bool IsDisposed
                                    IWebProxy proxy,
                                    TimeSpan idleTimeout,
                                    string identifier = default,
-                                   int sendBufferSizeBytes = AmqpConstants.TransportBufferSize,
-                                   int receiveBufferSizeBytes = AmqpConstants.TransportBufferSize,
+                                   int sendBufferSizeBytes = -1,
```

Comment:
Why remove the use of the constant?

Reply:
This should explicitly be -1 to force the platform defaults to be used.  The constant is the old T0 8192 value that doesn't perform well on linux.   We made the change to the options a little while back but missed the defaults here, which really just impact tests.